downstream/dsurf/_Surface.py

```diff
@@ -54,6 +54,45 @@ def enumerate(
         """Iterate over ingest times and values of retained data items."""
         return zip(self.lookup(), self._storage)
 
+    def ingest_multiple(
+        self: "Surface",
+        n_ingests: int,
+        item_getter: typing.Callable[[int], object],
+    ) -> typing.List[typing.Tuple[int, int]]:
+        """Ingest many data items.
+
+        Optimizes for the fact that if large amounts of data are ingested,
+        then we can skip assigning multiple objects to the same site, and 
+        simply iterate through sites that would get new data after items 
+        would be ingested.
+
+        Returns a list of any sites with new data, and the corresponding 
```

Comment:
Is a particular use case in mind for the return value? (ie for the hstrat impl?)

If not, I would remove so we have more flexibility with the interface in the future.

If it’s for testing, perhaps a similar functionality could be achieved by instrumenting the item getter in the testing environment to make sure certain items are gotten?